config/P-NUCLEO-LRWAN1_example_config.json

```diff
@@ -0,0 +1,67 @@
+{
+    "config": {
+        "lora-radio": {
+            "help": "Which radio to use (options: SX126X, SX1272, SX1276) -- See config/ dir for example configs",
+            "value": "SX1276"
+        },
+        "main_stack_size":     { "value": 4096 },
+
+        "lora-spi-mosi":       { "value": "NC" },
+        "lora-spi-miso":       { "value": "NC" },
+        "lora-spi-sclk":       { "value": "NC" },
```

Comment:
Why don't you set default values to D11.D12.D13 ?

Reply:
> Why don't you set default values to D11.D12.D13 ?

I used the same config structure as already used by the other two examples. An advantage of this is, that it is clear which board should work.
Of course every pin config could be done in the default section. When you thing that this is the better solution I'll move it.

Reply:
no preference, just question :-)